src/Utilities/regExpInsensitiveFilter.js

```diff
@@ -0,0 +1,47 @@
+// run a predicate function over each key of an object
+// const hasValue = f => o =>
+//  Object.keys(o).some(x => f(o[x]))
+const hasValue = f => o => {
+    let key;
+    for (key in o) {
+        if (f(o[key])) {return true;}
+
+        ;
```

Comment:
?

Reply:
This assert's that the object has the value we need

Reply:
Why is the `;` flying down here?

Reply:
Ah, I thought you are asking about the function itself. I removed these floating ;